lib/src/dartdoc_options.dart

```diff
@@ -243,11 +244,12 @@ class SnapshotCache {
     return snapshots[toolPath];
   }
 
-  void dispose() {
+  Future<void> dispose() {
+    _instance = null;
     if (snapshotCache != null && snapshotCache.existsSync()) {
-      snapshotCache.deleteSync(recursive: true);
+      return snapshotCache.delete(recursive: true);
     }
-    _instance = null;
+    return null;
```

Comment:
Can't you just delete this line now?

Reply:
The linter complains if I do that; I can leave the return in, or add a lint exclusion.